### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by DeXtroTip**
## Short Description:
Just a minor fix at emit event.

### CODE REVIEW

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -387,7 +387,7 @@ def _register_events(self):
         )
         self.event_manager.register_event(
             'next_egg_incubates',
-            parameters=('km_needed', 'distance_in_km',)
+            parameters=('km_needed', 'distance_in_km', 'eggs', 'eggs_inc')
```
Comment (mjmadsen) [inline pokemongo_bot/__init__.py]: The call to next_egg_incubates in incubate_eggs.py only sends one parameter currently.

Comment (DeXtroTip) [inline pokemongo_bot/__init__.py]: That's true, did not notice that.

**Comment by mjmadsen** [comment]
I would love if we could configure how often we display the hatch status and and listing all eggs in progress of being hatched, if you'd like to expand this.

**Comment by DeXtroTip** [comment]
@mjmadsen That's sounds a nice idea! I'll look into it.

**Comment by BreezeRo** [comment]
seems to use the two extra args now

`                self.emit_event(
                    'next_egg_incubates',
                    formatted='Next egg ({km_needed} km) incubates in {distance_in_km:.2f} km (Total eggs: {eggs}, Incubating: {eggs_inc})',
                    data={
                        'km_needed': self.used_incubators[0]['km_needed'],
                        'distance_in_km': km_left,
                        'eggs': len(self.eggs),
                        'eggs_inc': len(self.used_incubators)
                    }
                )`

**Comment by solderzzc** [comment]
@BreezeRo  Yes, this fixed no egg incubated log issue.

**Pull request merged**